fastcov.py

```diff
@@ -309,6 +321,12 @@ def exclMarkerWorker(fastcov_sources, chunk, exclude_branches_sw, include_branch
                             if start_line <= line_num <= end_line:
                                 del fastcov_data[key][line_num]
 
+                    for i in range(start_line, end_line):
```

Comment:
`i` is already declared and used by the outer loop. Better to use `line_num`
```suggestion
                    for line_num in range(start_line, end_line):
```